frontend/src/components/roadmap-survey/roadmap-survey.test.tsx

```diff
@@ -29,22 +29,36 @@ describe('RoadmapSurvey', () => {
   it('should render available and doing surveys in green scale', () => {
     render(<RoadmapSurvey groups={groups} />);
     expect(screen.getByTestId(`icon-${groups[0].surveys[0].name}`)).toHaveStyle(
-      'background: green'
+      'border: 4px solid green'
     );
     expect(screen.getByTestId(`icon-${groups[1].surveys[0].name}`)).toHaveStyle(
-      'background: green'
+      'border: 4px solid green'
     );
   });
 
   it('should render blocked surveys in gray scale', () => {
     render(<RoadmapSurvey groups={groups} />);
     expect(screen.getByTestId(`icon-${groups[2].surveys[0].name}`)).toHaveStyle(
-      'background: gray'
+      'border: 4px solid gray'
     );
   });
 
   it('should not render groups when not passed and show message', () => {
     render(<RoadmapSurvey groups={[]} />);
     expect(screen.getByText('No surveys to show =/')).toBeInTheDocument();
   });
+
+  it('should render survey icon background', () => {
+    render(<RoadmapSurvey groups={groups} />);
+    expect(screen.getByTestId(`icon-${groups[0].surveys[0].name}`)).toHaveStyle(
+      `background-image: url(${groups[0].surveys[0].icon_url})`
+    );
+  });
+
+  it('should render survey default icon background when dont have icon', () => {
```

Comment:
maybe `it should render "default survey icon" when the survey has no "icon_url"`